.github/actions/build_micro_service/action.yml

```diff
@@ -1,3 +1,8 @@
+# Required permissions:
+#
+# permissions:
+#   packages: write # for docker/build-push-action
```

Comment:
Why is this commented out?

Reply:
To document the required permissions of this action, every workflow that `use`s this action has to have at least this level of permissions.